manifest/v1alpha/budgetadjustment/budget_adjustment.go

```diff
@@ -43,10 +43,10 @@ type Spec struct {
 }
 
 type Filters struct {
-	Slos []Slo `json:"slos"`
+	SLOs []SLO `json:"slos"`
 }
 
-type Slo struct {
+type SLO struct {
```

Comment:
just one final thing, I think we should name it `SLORef`, similar to how we name such references in Role Binding and Alert Policy

Reply:
the tag for `slos` can stay as is, this is more of a code thing, when using completion it will be easier to have a single result for SLO instead of two